ple/games/donkeykong/Game.py

```diff
@@ -0,0 +1,210 @@
+__author__ = 'Batchu Vishal'
+import pygame
+import sys
+from pygame.locals import *
```

Comment:
If this is for the keys used consider specifying their names instead of importing everything